src/Access/IAccessStorage.cpp

```diff
@@ -151,6 +151,12 @@ std::vector<UUID> IAccessStorage::find(EntityType type, const Strings & names) c
 }
 
 
+std::optional<UUID> IAccessStorage::findOrGenerate(EntityType type, const String & name) const
```

Comment:
I've added the functions `IAccessStorage::login()` and `IAccessStorage::getIDOfLoggedUser()`, so we don't need the function `findOrGenerate()` anymore.

Reply:
Removed. Implemented the new `LDAPAccessStorage::loginImpl()` override.